### ISSUE TITLE
-output_format json is not respected with -score_type micromacro

### ISSUE THREAD

**Issue by danielhers** [flipz357/smatchpp#1]
I am running this command
```
                python -m smatchpp      -a $1 \
                        -b $2 \
                        -solver ilp \
                        -edges dereify \
                        -score_dimension main \
                        -score_type micromacro \
                        -log_level 20 \
                        -output_format json \
                        --bootstrap \
                        --remove_duplicates
```

And getting this output
```
-------------------------------
-------------------------------
---------Micro scores----------
-------------------------------
-------------------------------
{
    "main": {
        "F1": {
            "result": 27.12,
            "ci": [
                24.08,
                30.51
            ]
        },
        "Precision": {
            "result": 26.44,
            "ci": [
                22.81,
                30.18
            ]
        },
        "Recall": {
            "result": 27.84,
            "ci": [
                24.29,
                31.72
            ]
        }
    }
}


-------------------------------
-------------------------------
---------Macro scores----------
-------------------------------
-------------------------------
{
    "main": {
        "F1": {
            "result": 27.24,
            "ci": [
                23.99,
                30.41
            ]
        },
        "Precision": {
            "result": 28.61,
            "ci": [
                24.83,
                32.7
            ]
        },
        "Recall": {
            "result": 28.84,
            "ci": [
                24.92,
                32.74
            ]
        }
    }
}
```
This is not json :)

**Comment by flipz357** [comment]
Thanks Daniel, I made a fix for this in commit 2fbb13e 

Now it prints 100% json:

```
{
    "micro scores": {
        "main": {
            "F1": {
                "result": 75.18,
                "ci": [
                    74.52,
                    75.81
                ]
            },
            "Precision": {
                "result": 76.79,
                "ci": [
                    76.13,
                    77.42
                ]
            },
            "Recall": {
                "result": 73.63,
                "ci": [
                    72.88,
                    74.33
                ]
            }
        }
    },
    "macro scores": {
        "main": {
            "F1": {
                "result": 76.21,
                "ci": [
                    75.42,
                    77.01
                ]
            },
            "Precision": {
                "result": 77.79,
                "ci": [
                    77.0,
                    78.56
                ]
            },
            "Recall": {
                "result": 75.22,
                "ci": [
                    74.36,
                    76.07
                ]
            }
        }
    }
}
```

**Comment by danielhers** [comment]
Perfect, thank you!
